src/com/mercado/model/product/MeasureType.java

```diff
@@ -0,0 +1,17 @@
+package com.mercado.model.product;
+
+import com.mercado.model.BaseEntity;
+
+
+public class MeasureType extends BaseEntity {
+
```

Comment:
Remove the "new line" here, try keep consistency with prev. coding style as much as you can